user-console/source/org/pentaho/mantle/public/themes/crystal/mantleCrystal.css

```diff
@@ -1283,10 +1283,19 @@ a:active, a:hover {
 }
 
 .users-roles-list {
-  height: 432px;
+  height: 100%;
   width: 200px;
 }
 
+/* targetting IE 10 only */
+.IE10 .pentaho-tab-panel {
+    height:541px!important;
```

Comment:
Even though it's usually used with a space, it works either way. In this case, the change is reflected on the styles being applied so it's working properly.